serving/docker/partition/neo_utils.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
Can you rename this file to be `sm_neo_utils.py` to match the naming convention of `sm_neo_trt_llm_partition.py`

Reply:
Done